app/assets/stylesheets/palettes/_base.scss

```diff
@@ -3,3 +3,185 @@ $color_disabled_background: rgba(0, 0, 0, 0.2); // disable element background
 $color_facet_panel_background: rgba(255, 255, 255, 0.3); // facet panel background
 $color_topnav: #eee;
 $color_topnav_background: #222;
+
```

Comment:
Thinking these definitions should be moved to `stylesheets/layouts/_base.scss`

Reply:
Or maybe into a separate `_bs4_compat.scss` and included there?

Reply:
a bs4 compat sheet sounds great. will do!